src/UXClient/Components/LineChart/LineChart.ts

```diff
@@ -720,9 +905,10 @@ class LineChart extends ChartComponent {
                             y: aggY,
                             visible: visibleYAxis
                         };
+                        var splitByColors = Utils.createSplitByColors(this.chartComponentData.displayState, aggKey, this.chartOptions.keepSplitByColor);
         
                         Object.keys(this.chartComponentData.timeArrays[aggKey]).forEach((splitBy: string, j: number) => {
-
+                            this.colorMap[aggKey + "_" + splitBy] = splitByColors[j];
```

Comment:
oh yeah i thought about this while driving in, we never place stuff like this in the dom correct?  IIRC we did a pass where ids and classnames were not reallly bound to names of the data passed in, which we should enforce in case people have really trash names